### ISSUE TITLE
"MongoDB" is not a correct storage driver

### ISSUE THREAD

**Issue by lifehome** [Cog-Creators/Red-DiscordBot#2589]
# Other bugs

#### What were you trying to do?

Just update the bot.

#### What were you expecting to happen?

Nothing.

#### What actually happened?

`MongoDBV2` popped up, replaced `MongoDB`. But there is no handler for `MongoDB`, not even raising exceptions.

```
Traceback (most recent call last):
  File "/opt/RedBot/bin/redbot", line 11, in <module>
    load_entry_point('Red-DiscordBot==3.0.2', 
  File "/opt/RedBot/lib/python3.7/site-packages/
    red = Red(cli_flags=cli_flags, description=description, 
  File "/opt/RedBot/lib/python3.7/site-packages/redbot/core/
    self.db = Config.get_core_conf(force_registration=True)
  File "/opt/RedBot/lib/python3.7/site-packages/redbot/core/
    driver_name, "Core", "0", data_path_override=core_path, 
  File "/opt/RedBot/lib/python3.7/site-packages/redbot/core/
    raise RuntimeError("Invalid driver type: '{}'".format(type))
RuntimeError: Invalid driver type: 'MongoDB'
```

#### How can we reproduce this issue?

Just update a bot version prior the MongoDBV2 changes to latest dev and you will see.

**Comment by tekulvw** [comment]
This isn't actually a bug, due to the latest Mongo rewrite we had to differentiate between the "old" mongo and the "new" mongo drivers. #2579 will provide conversion capabilities back between the old mongo and JSON. Instructions on how to update to 3.1 if you are a Mongo user will be provided in the docs but do not exist yet.

**Comment by Stonedestroyer** [comment]
This is fixed by, https://github.com/Cog-Creators/Red-DiscordBot/pull/2594.

**Comment by tekulvw** [comment]
Actually this is fixed by #2603 where a more descriptive error message is printed.

**Comment by tekulvw** [comment]
Fixed with #2603
